ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/MockCrypto.hs

```diff
@@ -36,3 +40,29 @@ instance HashAlgorithm h => Crypto (TPraosMockCrypto h) where
 instance HashAlgorithm h => TPraosCrypto (TPraosMockCrypto h)
 
 type Block h = ShelleyBlock (TPraosMockCrypto h)
+
+-- | Like 'TPraosMockCrypto' except mocks fewer components in order to be
```

Comment:
Should this crypto bundle be relocated to `ouroboros-consensus-cardano/src/Test/*`? (That file hierarchy does not yet exist.)

Reply:
Done, thanks!